modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -179,24 +240,32 @@ public void run() {
 
                     Lease lease = leaseTracker.getLease(grpId);
 
+                    if (!lease.isAccepted()) {
+                        LeaseAgreement agreement = leaseConciliator.conciliated(grpId);
+
+                        if (agreement.isAccepted()) {
+                            acceptLeasInMetaStorage(grpId, lease);
+                        }
+                    }
```

Comment:
There should be a check that the agreement contains the same lease.

Reply:
Here, checking a lease is not required because, because we cannot update the corresponded entry in Meta storage if it is not actual.